py/server/test_helper/__init__.py

```diff
@@ -75,6 +74,11 @@ def start_jvm_for_tests(jvm_props: Dict[str, str] = None):
         _JPythonScriptSession = jpy.get_type("io.deephaven.integrations.python.PythonDeephavenSession")
         py_dh_session = _JPythonScriptSession(_j_test_update_graph, py_scope_jpy)
 
+        _JUpdatePerformanceTracker = jpy.get_type("io.deephaven.engine.table.impl.perf.UpdatePerformanceTracker")
+        exec_ctx = py_dh_session.getExecutionContext().open()
+        _JUpdatePerformanceTracker.start(_j_test_update_graph)
+        exec_ctx.close()
+
```

Comment:
I'm unsure why this is needed for Python tests.

Reply:
I was wondering the same thing.